src/js/blue-button/containers/BlueButtonApp.jsx

```diff
@@ -9,7 +9,7 @@ function AppContent({ children, isDataAvailable }) {
   const unregistered = isDataAvailable === false;
   let view;
 
-  if (unregistered) {
+  if (!unregistered) {
```

Comment:
I don't think this is correct. If the user is unregistered (when `isDataAvailable` is specifically `false`), it should show the error message.

If you were getting it after being logged in, it could be because the account doesn't have access to this service, or Blue Button hasn't been added to the list of services for the user on the API side yet. I believe it's the latter, since I don't see it within the list of services:

* https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/backend_services.rb
* https://github.com/department-of-veterans-affairs/vets-api/blob/master/docs/api/user.md#response

So we should probably get someone to add it there. For now, in order to test, all I was doing was removing the `RequiredLoginView` wrapper, since we haven't hooked up to the API yet.